genstudio-mlr-claims-app/package.json

```diff
@@ -1,51 +1,56 @@
-        "core-js": "^3.6.4",
-        "node-fetch": "^2.6.0",
-        "node-html-parser": "^5.4.2-0",
-        "react": "^16.13.1",
-        "react-dom": "^16.13.1",
-        "react-error-boundary": "^1.2.5",
-        "react-router-dom": "^6.3.0",
-        "regenerator-runtime": "^0.13.5"
-    },
-    "devDependencies": {
-        "@babel/core": "^7.8.7",
-        "@babel/plugin-transform-react-jsx": "^7.8.3",
-        "@babel/polyfill": "^7.8.7",
-        "@babel/preset-env": "^7.8.7",
-        "@openwhisk/wskdebug": "^1.3.0",
-        "@types/node": "^16.0.0",
-        "@types/react": "^16.14.62",
-        "@types/react-dom": "^16.9.25",
-        "eslint": "^8",
-        "eslint-plugin-jest": "^27.2.3",
-        "jest": "^27.2.4",
-        "typescript": "^4.9.0"
-    },
-    "scripts": {
-        "test": "jest --passWithNoTests ./test",
-        "e2e": "jest --collectCoverage=false --testRegex ./e2e",
-        "lint": "eslint --ignore-pattern web-src --no-error-on-unmatched-pattern test src actions",
-        "lint:fix": "npm run lint -- --fix",
-        "type-check": "tsc --noEmit",
-        "type-check:watch": "tsc --noEmit --watch",
-        "build": "tsc"
-    },
-    "description": "GS Create UIX Reference App",
-    "aio-app-builder-templates": [
-        "@adobe/aem-cf-editor-ui-ext-tpl"
-    ]
+  "name": "genstudio-create-validation",
+  "version": "0.2.0",
+  "private": true,
+  "dependencies": {
+    "@adobe/aem-cf-editor-ui-ext-tpl": "^2.1.0",
+    "@adobe/aio-sdk": "^6",
+    "@adobe/exc-app": "^0.2.21",
+    "@adobe/generator-add-action-generic": "^1.0.1",
+    "@adobe/genstudio-uix-sdk": "^1.6.5",
+    "@adobe/react-spectrum": "^3.4.0",
+    "@adobe/uix-guest": "^1.0.0",
+    "@react-spectrum/list": "^3.0.0-rc.0",
+    "@spectrum-icons/workflow": "^3.2.0",
+    "chalk": "^4",
+    "core-js": "^3.6.4",
+    "node-fetch": "^2.6.0",
+    "node-html-parser": "^5.4.2-0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-error-boundary": "^1.2.5",
+    "react-router-dom": "^6.3.0",
+    "regenerator-runtime": "^0.13.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.8.7",
+    "@babel/plugin-transform-react-jsx": "^7.8.3",
+    "@babel/polyfill": "^7.8.7",
+    "@babel/preset-env": "^7.8.7",
+    "@openwhisk/wskdebug": "^1.3.0",
+    "@types/node": "^16.0.0",
+    "@types/react": "^16.14.62",
+    "@types/react-dom": "^16.9.25",
+    "eslint": "^8",
+    "eslint-plugin-jest": "^27.2.3",
+    "jest": "^27.2.4",
+    "typescript": "^4.9.0"
+  },
+  "scripts": {
+    "test": "jest --passWithNoTests ./test",
+    "e2e": "jest --collectCoverage=false --testRegex ./e2e",
+    "lint": "eslint --ignore-pattern web-src --no-error-on-unmatched-pattern test src actions",
+    "lint:fix": "npm run lint -- --fix",
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "tsc --noEmit --watch",
+    "build": "tsc"
+  },
+  "description": "GS Create UIX Reference App",
+  "aio-app-builder-templates": [
```

Comment:
oops accidentally have some of the stuff in here from what I was working on for the runtime app